recipe/meta.yaml

```diff
@@ -176,12 +188,14 @@ requirements:
     - llvmdev
     - clangdev
     - libclang
+    - pcre2
```

Comment:
why did you add pcre2?

Reply:
i'm trying to build qt without pcre2 on the server. i'll let you know the progress tomorrow i guess.

Reply:
Hi @hmaarrfk,
I first tried building it without pcre2. This failed with the linker error in https://github.com/conda-forge/qt-feedstock/pull/181#issuecomment-770462172

I thought I might resolve this linker issue by using the `system-pcre` flag instead of the `qt-pcre` flag. This requires pcre2 with this patch: https://github.com/conda-forge/pcre2-feedstock/pull/26

However, the linker error is still the same. The linker error relates to pcre(1) rather than pcre2, so using the `system-pcre` flag instead of the `qt-pcre` flag doesn't make any difference. It must be something else.

Re CDT/libgthread: I don't understand enough about this to have an informed opinion. My best guess is that somewhere within the QT webengine it uses the correct linking command to pull in pcre, while it doesn't do it by just using QT core.

I uploaded the built qt package to https://anaconda.org/robostack/qt

Installing that in an environment and then running the tests might help you understand what's going on.

Reply:
thanks. i killed the build.